packages/widget/src/controllers/transfers/fungible-token-transfer.ts

```diff
@@ -47,6 +54,7 @@ export class FungibleTokenTransferController implements ReactiveController {
   public supportedSourceNetworks: Domain[] = [];
   public supportedDestinationNetworks: Domain[] = [];
   public supportedResources: Resource[] = [];
+  public fee?: EvmFee;
```

Comment:
Can the fee be undefined? My understanding that it's always should be defined, in case the fee is not applicable it will be zero instead. @mpetrunic can you help me here to understand what are the other factors that would influence whether the fee exists or not?

Reply:
You need to set all the fields to be able to calculate and set fee in this field. Fee depends on route/resource chosen and amount